polyhost/device/poly_kybd.py

```
@@ -70,8 +75,13 @@ def connect(self):
             # All retries exhausted — HID handle is stale after a reset/reflash;
             # re-enumerate so the new USB path is picked up.
             self.log.warning("Re-enumerating HID after %d failed attempts...", retries)
-            self.hid = HidHelper(self.device_settings)
-            self.serial = SerialHelper(self.device_settings)
+            try:
+                self.hid = HidHelper(self.device_settings)
+                self.serial = SerialHelper(self.device_settings)
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)